packages/core/src/components/HeaderIconLinkRow/IconLinkVertical.tsx

```diff
@@ -37,52 +39,54 @@ const useIconStyles = makeStyles(theme => ({
   disabled: {
     color: 'gray',
   },
+  primary: {
+    color: theme.palette.primary.main,
+  },
+  secondary: {
+    color: theme.palette.secondary.main,
+  },
   label: {
     fontSize: '0.7rem',
     textTransform: 'uppercase',
     fontWeight: 600,
     letterSpacing: 1.2,
   },
-  linkStyle: {
-    color: theme.palette.secondary.main,
-  },
 }));
 
 export function IconLinkVertical({
   icon = <LinkIcon />,
   href = '#',
   disabled = false,
-  action,
-  ...props
+  color = 'primary',
```

Comment:
AFAIK, no the primary is blue. Thing is it's a "requirement" for us to allow the pager duty link to be different, since this is how it used to look like before. We could change that as well and deal with the stakeholders, but I think it's probably ok to accept 2 variants (but not more, else this can become a carnival).

Reply:
Ah I read wrong it seems